src/pke/unittest/utbfvrns/UnitTestBFVrnsSerialize.cpp

```diff
@@ -33,27 +33,27 @@
 #include "UnitTestSer.h"
 #include "gtest/gtest.h"
 
+#include "cryptocontexthelper.h"
 #include "scheme/bfvrns/bfvrns-ser.h"
 
 using namespace std;
 using namespace lbcrypto;
 
 class UTPKESer : public ::testing::Test {
- protected:
-  void SetUp() {}
+protected:
+    void SetUp() {}
 
-  void TearDown() {
-    CryptoContextImpl<DCRTPoly>::ClearEvalMultKeys();
-    CryptoContextImpl<DCRTPoly>::ClearEvalSumKeys();
-    CryptoContextFactory<DCRTPoly>::ReleaseAllContexts();
-  }
+    void TearDown() {
+        CryptoContextImpl<DCRTPoly>::ClearEvalMultKeys();
+        CryptoContextImpl<DCRTPoly>::ClearEvalSumKeys();
+        CryptoContextFactory<DCRTPoly>::ReleaseAllContexts();
+    }
 };
 
-CryptoContext<DCRTPoly> GenerateTestDCRTCryptoContext(const string& parmsetName,
-    usint nTower,
-    usint pbits) {
-    CryptoContext<DCRTPoly> cc =
-        CryptoContextHelper::getNewDCRTContext(parmsetName, nTower, pbits);
+CryptoContext<DCRTPoly> GenerateTestDCRTCryptoContext(const string& parmsetName, usint nTower, usint pbits) {
+    // TODO (dsuponit): getNewDCRTContext() should be replaced with a context generator function call
```

Comment:
Still needs doing?

Reply:
I added comments as I didn't want my finding to be lost. All those things are to be corrected/changed later.